ui/page/staking/stake_info.go

```diff
@@ -78,7 +78,7 @@ func (pg *Page) stakePriceSection(gtx C) D {
 										}),
 										layout.Rigid(func(gtx C) D {
 											return layout.Center.Layout(gtx, func(gtx C) D {
-												if pg.dcrWallet.IsSyncing() || pg.dcrWallet.IsRescanning() || !pg.isTicketsPurchaseAllowed() {
+												if !pg.dcrWallet.IsSynced() || pg.dcrWallet.IsRescanning() || !pg.isTicketsPurchaseAllowed() {
```

Comment:
`pg.dcrWallet.IsSyncing()` and `pg.dcrWallet.IsSynced()` can both be false. I think checking for synced is correct.